scripts/kanzus_dev

```diff
@@ -0,0 +1,219 @@
+
+
+        if event['cbf_num'] % n_batch_transfer == 0 or event['cbf_num'] == n_initial_transfer:
+            start_transfer_flow(event)
+
+        if event['cbf_num'] % n_batch_stills == 0:
+            start_stills_flow(event)
+
+        if event['cbf_num'] % n_batch_publish == 0:
+            start_publish_flow(event)
+
+        if event['cbf_num'] % n_batch_prime == 0:
+            start_prime_flow(event)
+
+    if 'int_list.txt' in event_file:
+       event = parse_int_event(event_file)
+       
+       if event['total_ints'] % n_batch_prime == 0:
+           start_prime_flow(event)
+
+
+def start_transfer_flow(event):
+    ###RunFlow
+    label = 'SSX_Transfer_{}_{}'.format(event['chip_name'],event['cbf_num'])
+    flow = data_transfer_flow.run_flow(flow_input=base_input,label=label)
+    print(label)
+    print("URL : https://app.globus.org/runs/" + flow['action_id'] + "\n")
+    ###
+
+def start_stills_flow(event):
+    ###RunFlow
+    label = 'SSX_Stills_{}_{}'.format(event['chip_name'],event['cbf_num'])
+    flow = stills_flow.run_flow(flow_input=flow_input, label=label)
+    print(label)
+    print("URL : https://app.globus.org/runs/" + flow['action_id'] + "\n")
+    ###
+
+
+def start_publish_flow(event):
+    ###RunFlow
+    label = f'SSX_Plot_{}_{}'.format(event['chip_name'],event['cbf_num'])
+    flow = publish_flow.run_flow(flow_input=base_input,label=label)
+    print(label)
+    print("URL : https://app.globus.org/runs/" + flow['action_id'] + "\n")
+    ###
+
+def start_prime_flow(event):                                   
+    ###RunFlow
+    label = 'SSX_Prime_{}_{}'.format(event['chip_name'],event['cbf_num'])                                                                                                                                    
+    flow = prime_flow.run_flow(flow_input=base_input, label=label)
+    print(label)
+    print("URL : https://app.globus.org/runs/" + flow['action_id'] + "\n")
+    ###
+
+# Arg Parsing
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('localdir', type=str, default='.')
+    parser.add_argument('--datadir', type=str, 
+        default='/eagle/APSDataAnalysis/SSX/random_start')
+    parser.add_argument('--deployment','-d', default='raf-prod', help=f'Deployment configs. Available: {list(deployment_map.keys())}')
+    return parser.parse_args()
+
+##Gather deployment specific variables
+from gladier_kanzus.deployments import deployment_map
+
+##Import relevant flows
+from gladier_kanzus.flows import TransferFlow
+from gladier_kanzus.flows import StillsFlow
+from gladier_kanzus.flows import PublishFlow
+from gladier_kanzus.flows import PrimeFlow
+
+
+
+if __name__ == '__main__':
+
+    args = parse_args()
+
+    local_dir = args.localdir
+    data_dir = args.datadir
+
+    n_initial_transfer = 512
+    n_batch_transfer = 2048
+    n_batch_stills = 512
+    n_batch_publish =  2048
+    n_batch_prime =  2000
```

Comment:
It would be good to document what each of these do. Should they be configurable parameters?

Reply:
Adding comment on it now.